modules/ROOT/pages/single-sign-on-with-oidc.adoc

```diff
@@ -150,50 +151,66 @@ The expected format for Keys and certificates is PEM (with or without the commen
   *** `token-store` indicates where the account information of the logged in user is stored on the client (Bonita bundle). By default it is `session` (HTTP session). `cookie` is also supported by the client adapter but it is not recommended to change it as all Bonita OIDC integration has been validated with the `session` setting.
-  *** `verify-token-audience` can be set to true to verify that the bearer token audience contains the client name (`resource ` property).
+  *** `verify-token-audience` can be set to `true` to verify that the JWT tokens audience (`aud` claim) contains the client Id (`resource` property).
+  *** `redirect-rewrite-rules` can be used to modify the path section of the redirect URI sent to the OIDC provider with the authentication or code exchange requests. By default the URI sent is the request URI as seen by Bonita OIDC module, but it may not be the URI the OIDC provider needs to redirect to if Bonita is behind a reverse proxy for example. The value of this property is an object where each attribute key is a regular expression to which the Redirect URI path is to be matched and the attribute value is the replacement String. The other parts of the URI like the protocol, domain, port or query string cannot be modified. Here is an example:
++
+----
+  "redirect-rewrite-rules" : {
+    "^(.*)$" : "/portal/homepage"
+  }
+----
 
 [NOTE]
 ====
 
 More configuration options can be found in https://www.keycloak.org/docs/latest/securing_apps/#_java_adapter_config[Keycloak official documentation]
 ====
 
-. If your Identity Provider is corectly configured (see the section _Configure the Identity Provider_), you are done.
+. If your Identity Provider is correctly configured (see the section _Configure the Identity Provider_), you are done.
 Then you can try to access a portal page, an app page or a form URL (or just `http://<host>:<port>/bonita[?tenant=<tenantId>]`) and make sure that you are redirected to your Identity Provider to log in (unless you are already logged in). +
-Note that if you try to access `http://<bundle host>:<port>/bonita/login.jsp`, then you won't be redirected as this page still needs to be accessible in order for the tenant administrator (or another user if you set the property `oidc.auth.standard.allowed` to true) to be able to log in without an account on the Identity Provider.
+Note that if you try to access `http://<bundle host>:<port>/bonita/login.jsp`, then you won't be redirected as this page still needs to be accessible in order for the tenant administrator (or another user if you set the property `oidc.auth.standard.allowed` to `true`) to be able to log in without an account on the Identity Provider.
```

Comment:
I don't understand what this sentence means.

Reply:
clarified